NOTES.md

```diff
@@ -1,3 +1,105 @@
+# Release 2.1.0-beta (Das Boot)
+
+-- issues up-to-date with the 2.1 milestone as of 2015-07-06 23:00 UTC
+-- contributors " " next as of 3dac994 and docs:master as of 0b661c4
+
+Release highlight:
+
+* Automatic failover
+
+Read the [blog post][2.1-beta] for more details.
+
+[2.1-beta]: http://rethinkdbblog.com/blog/2.1-beta/
+
+## Compatibility ##
+
+This beta release does not include automatic migration of data
+directories from older versions of RethinkDB. However the final 2.1.0
+version will include that feature for versions of RethinkDB down to
+1.14.0. Upgrading directly from RethinkDB 1.13 will no longer be
+supported.
+
+### API-breaking changes ###
+
+* `.split('')` now treats the input as UTF8 instead of an array of bytes.
+* `null` values in compound index are no longer discarded.
+* The new `read_mode` optional argument replaces `use_outdated`
+* The format of the system tables has changed
+
+## New features ##
+
+* Server
+ * Added automatic failover and semi-lossless rebalance based on raft (#223)
+ * Backfills are now interuptible and reversible (#3886, #3885)
+ * `table.reconfigure()` now works even if some servers are disconnected (#3913)
+ * Replicas can now be marked as voting or non-voting (#3891)
+ * Table reconfigurations can now be applied without consensus (#3893)
```

Comment:
Since there was no notion of consensus in the old system apart from what's already covered by "`table.reconfigure()` now works even if some servers are disconnected (#3913)",
How about this formulation instead?
Added an emergency repair feature to restore table availability if consensus is lost (#3893)